hnn_core/gui.py

```diff
@@ -188,11 +191,11 @@ def update_plot_window(variables, plot_out, plot_type):
         return
 
     with plot_out:
-        if plot_type['new'] == 'spikes':
+        if plot_type['new'] == 'spikes': # BUG: got warning from matplotlib and numpy
```

Comment:
what was the warning you got?

Reply:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/11160442/176448014-0d3a0d5e-338a-42a8-9c6a-d3d6ec280890.png">
I thinks it's because there're no data. Just added some extra conditions in case the layout is broken